modules/core/src/lib/deck.ts

```diff
@@ -117,6 +117,9 @@ export type DeckProps<ViewsT extends ViewOrViews = null> = {
   /** Use an existing luma.gl GPU device. @note If not supplied, a new device will be created using props.deviceProps */
   device?: Device | null;
 
+  /** Supply adapters to use when a new device is created */
+  adapters?: Adapter[];
```

Comment:
I thought adapters would be added via `luma.registerAdapters`?

Reply:
> I thought adapters would be added via `luma.registerAdapters`?

registerAdapters is still supported, but generally I find that global registration creates as many problems as it solves, so added the capability to specify adapters explicitly.